src/Features/CSharp/CSharpFeaturesResources.resx

```diff
@@ -228,6 +228,9 @@
   <data name="Deprecated" xml:space="preserve">
     <value>deprecated</value>
```

Comment:
Do you mind capitalizing "deprecated" as part of this change as well? I agree with @dsaf that the lowercasing makes it seem like a keyword instead of an attribute.